pkgs/development/python-modules/jep/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, python, fetchPypi, buildPythonPackage, numpy, jre, numpySupport ? true }:
+
+buildPythonPackage rec {
+  pname = "jep";
+  version = "3.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "fM0lqSoZo5FQTnZpQBkL3NTWs6hIjKSjrcjrjNWBwMs=";
+  };
+
+  buildInputs = [ jre ];
+  propagatedBuildInputs = lib.optional numpySupport numpy;
+
+  preCheck = lib.optional numpySupport ''
+    export PYTHONHOME="${python.withPackages(_: [ numpy ])}"
+  '';
+
+  postFixup = ''
+    cat <<EOF > $out/bin/jep
+    #!/bin/sh
+
+    export NIX_PYTHONPATH="$out/${python.sitePackages}:\$NIX_PYTHONPATH"
```

Comment:
Is there a better way to do this?
Reason for this being there is that the jep python module needs to be on the path for this to work correctly. Without this it breaks when users install this packages using `pythonPackages.jep`